discord/colour.py

```diff
@@ -333,6 +333,26 @@ def nitro_pink(cls: Type[CT]) -> CT:
         .. versionadded:: 2.0
         """
         return cls(0xf47fff)
+    
+    @classmethod
+    def embed_background(cls: Type[CT], theme: str = "dark") -> CT:
+        """A factory method that returns a :class:`Color` corresponding to the embed colors on discord clients, with a value of 
+        ``0x2F3136`` (dark)
+        ``0xf2f3f5`` (light)
+        ``0x000000`` (amoled).
+        
+        .. versionadded:: 2.0
+        """
+        themes_cls = {
+            "dark": 0x2F3136,
+            "light": 0xf2f3f5,
+            "amoled": 0x000000
```

Comment:
```suggestion
            "amoled": 0x000000,
```

Reply:
could you explain that to me? 😅

Reply:
Trailing commas are not necessary but I think it's good styling. They also remove the need for modifying 2 lines to add something to the list.

Reply:
Ah, same like c# then

Reply:
It's part of PEP-8 to make git blame and just general git updates nicer, because by adding the comma, you update the whole line, meaning it shows up in commits. Completely with Dorukyum here